go.mod

```diff
@@ -39,7 +41,7 @@ require (
 	github.com/spf13/viper v1.15.0
 	github.com/stretchr/testify v1.8.2
 	github.com/tigrisdata/tigris-client-go v1.0.0-beta.25
-	github.com/typesense/typesense-go v0.6.2
+	github.com/tigrisdata/typesense-go v0.6.2-beta.6
```

Comment:
Because their go library was last released in December, it’s behind in features and doesn’t support vector search feature for example

Reply:
Ok that makes sense.

Reply:
That's right, we are mainly using the forked version so that we can control their open API spec and regenerate the client. I also found out that their latest open API spec will break us as it has some incompatible changes i.e. the API changes are not tied to the server version changes so one another reason to fork it.